store/tikv/metrics/metrics.go

```diff
@@ -394,6 +404,21 @@ func initMetrics(namespace, subsystem string) {
 			Buckets:   prometheus.ExponentialBuckets(0.000005, 2, 30), // 5us ~ 2560s
 		})
 
+	TiFlashExecuteErrorCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "tiflash_execute_error_total",
+			Help:      "Counter of TiFlash execute errors.",
+		})
+
+	TiFlashExecuteSuccCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "tiflash_execute_total",
+			Help:      "Counter of TiFlash execute successfully.",
+		})
```

Comment:
```suggestion
	TiFlashExecuteErrorCounter = prometheus.NewCounterVec(
		prometheus.CounterOpts{
			Namespace: namespace,
			Subsystem: subsystem,
			Name:      "tiflash_requests_total",
			Help:      "Counter of TiFlash requests.",
		}, []string{LblResult})

```

Reply:
Use 1 metric counter is much easer to calculate the `Number of “Failure query” / total queries`

Reply:
Oh, wait, The original demand is calculate the ** failure query**, not the `failure tiflash cop request`.

so the metric maybe rename to following is better.

```go
	TiFlashQueryTotalCounter = prometheus.NewCounterVec(
		prometheus.CounterOpts{
			Namespace: "tidb",
			Subsystem: "server",
			Name:      "tiflash_query_total",
			Help:      "Counter of TiFlash queries.",
		}, []string{LblResult})

```

And the metrics should put in `metrics/server.go` file.

Reply:
I suspect whether the "total queries" are counted correctly if we use an existing metrics. For example, does it count DDL? Does it count prepared statements? etc. I think @jyz0309 should take a look about this first before making this change, make sure that "total queries" to be replaced is exactly the same as the counter we added here.